events/helper/outbound_connection.go

```diff
@@ -0,0 +1,20 @@
+package helper
+
+import (
+	"net"
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(OutboundConnection)
+
+func OutboundConnection(h events.Helper) error {
+	conn, err := net.Dial("udp", "8.8.8.8:53")
+	if err != nil {
+		return err
+	}
+	defer conn.Close()
+	if _, err := conn.Write([]byte("Sent by event-generator")); err != nil {
+		return err
+	}
```

Comment:
re:
> Sending written data is not required to trigger this rule; the important thing is a successful connection. I included this part (sending written data) in action, to establish a connection only. If you prefer, we can remove this part.

I've no strong opinion. I'd avoid bothering 3rd-party service, if possible. 
@FedeDP wdyt?

Reply:
Agree, since the rule triggers without the data being actually written, we should just skip this part.

Reply:
@FedeDP @leogr made changes.

Reply:
hey @h4l0gen

have you tried the latest implementation? does it still trigger the rule? :thinking:

Reply:
which part of this macro condition is meeting?

```
- macro: outbound
  condition: >
    (((evt.type = connect and evt.dir=<) or
      (evt.type in (sendto,sendmsg) and evt.dir=< and
       fd.l4proto != tcp and fd.connected=false and fd.name_changed=true)) and
     (fd.typechar = 4 or fd.typechar = 6) and
     (fd.ip != "0.0.0.0" and fd.net != "127.0.0.0/8" and not fd.snet in (rfc_1918_addresses)) and
     (evt.rawres >= 0 or evt.res = EINPROGRESS))
```

Reply:
Hii @leogr. Yes rule triggered successfully after changes done.
![Screenshot from 2024-05-01 03-45-59](https://github.com/falcosecurity/event-generator/assets/117383853/21138034-1470-4f1f-9759-525b84091f99)

Reply:
- As we used `net.Dial` method, which performs outbound connection hence ```(evt.type = connect and evt.dir=<)``` is satisfied. 
- UDP connectuon is used. 
- Using `example.net` not RFC 1918 addresses (private IP range)